### ISSUE TITLE
populartimes values

### ISSUE THREAD

**Issue by sherwinhlee** [m-wrzr/populartimes#44]
I can't find information on this but what exactly are 'populartimes' values? More importantly, is this normalized for all Places?

**Comment by rmateus** [comment]
No, it is not normalized for all Places. It is normalized to the highest average weekly attendance for the respective Place only.

**Comment by sherwinhlee** [comment]
Okay, thanks. So to be clear, each element in the 24-length vector represents actual attendance?

**Comment by Porter97** [comment]
The values themselves are derived from a combination of google searches, google maps app location data, and local traffic data. This data is then used on a per location basis and gives a weekly (by hour and by day) reading for how busy that particular location is on a scale of 1-100. (1 being the least busy and 100 being the busiest a particular location gets).

**Comment by m-wrzr** [comment]
@Porter97 awesome description, can I add this to the readme or could you make a quick PR?
